gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/HttpDriverIntegrateTest.java

```diff
@@ -97,7 +121,8 @@ public void shouldSubmitBytecodeWithGraphBinary() throws Exception {
                 .create();
         try {
             final GraphTraversalSource g = traversal().withRemote(DriverRemoteConnection.using(cluster));
-            assertEquals("2", g.inject("2").toList().get(0));
+            final String result = g.inject("2").toList().get(0);
```

Comment:
Why was this change necessary?

Reply:
not necessary, consequences of debugging